xpsi/EnsembleSampler.py

```diff
@@ -172,7 +172,7 @@ def _nd_ball(self, moments):
         else:
             raise PriorError
 
-        moments = list(map(list, zip(*moments)))
+        moments = list(map(list, list(zip(*moments))))
```

Comment:
Both `list(map(list, zip(*moments)))` and `list(map(list, list(zip(*moments))))` work.
@dhuppenkothen Do you have a sense of which should be use ?